### PR TITLE
[DevTools] Complete moving renderRelayTree into its own file.

### PR DESCRIPTION
**Pull request by alloy**
This was super duper confusing. My bad! 🙈

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit d227ad1e4bbc936432029170374a2ecfda86d85e

https://deploy-preview-1392--artsy-reaction.netlify.com

File: src/DevTools/MockRelayRenderer.tsx [hunk 1]
```diff
@@ -19,10 +18,13 @@ export interface MockRelayRendererProps {
...
- * Renders a tree of Relay containers.
+ * Renders a tree of Relay containers with a mocked local instance of the
+ * metaphysics schema.
  *
- * Because Relay works asynchronously _and_ a tree may contain nested
- * `QueryRenderer` components, you should usually use {@link renderRelayTree}.
+ * @note
+ * Use this component in storybooks, but not tests. Because Relay works
+ * asynchronously _and_ a tree may contain nested `QueryRenderer` components,
+ * for tests you should usually use {@link renderRelayTree}.
```
Comment (alloy) [inline src/DevTools/MockRelayRenderer.tsx]: The below example code should probably show a story rather than a test 🤔

**Comment by alloy** [comment]
@xtina-starr is going to use this today, so re-assigning

**Comment by alloy** [comment]
We didn’t need to do this right now after all, so leaving it for you @damassi

**Comment by damassi** [comment]
LGTM 👍

**Pull request merged**